tox.ini

```diff
@@ -14,7 +14,7 @@ isolated_build = true
 
 [testenv]
 # Pass through the following environment variables which may be needed for the CI
-passenv = HOME WINDIR LC_ALL LC_CTYPE CC CI TRAVIS
+passenv = HOME,WINDIR,LC_ALL,LC_CTYPE,CC,CI,TRAVIS
```

Comment:
Tox changed the way it parses environment variables...comma delimited now.

Reply:
Yay...